### PR TITLE
[DOC] niekvanderlaan contrib

### PR DESCRIPTION
**Pull request by fkiraly**
@niekvanderlaan contributed a fix in https://github.com/alan-turing-institute/sktime/pull/1868 which remained incomplete, but got eventually completed by @k1m190r and merged.

Most work was already done by @niekvanderlaan, except linting.

### CODE REVIEW

**Review by ltsaprounis — DISMISSED** [review]
Seems fair. 
Shall we wait for a thumbs up from @niekvanderlaan before we merge?

**Comment by fkiraly** [comment]
I guess?

**Pull request merged**
